client/package.json

```diff
@@ -19,7 +19,8 @@
     "print-phantomjs-version": "phantomjs --version",
     "start-phantomjs": "phantomjs --webdriver=8910",
     "styleguide": "start-storybook -p 9001",
-    "deploy-storybook": "storybook-to-ghpages"
+    "deploy-storybook": "storybook-to-ghpages",
+    "test": "mocha --reporter mocha-circleci-reporter --require babel-register --require ignore-styles --require ./mochaConfig.js './app/components/composites/ListingCard/ListingCard.story.js'"
```

Comment:
Maybe we could move babel-register etc. to mochaConfig.js
See: https://github.com/lelandrichardson/enzyme-example-mocha/blob/master/test/.setup.js